src/FFMpeg/FFMpeg.php

```diff
@@ -93,7 +93,14 @@ public function open($pathfile)
             throw new RuntimeException(sprintf('Unable to probe "%s".', $pathfile));
         }
 
-        if (0 < count($streams->videos())) {
+        $hasVideo = false;
+        foreach ($streams->videos() as $videoStream) {
+            $videoDisposition = $videoStream->get('disposition');
+            if ($videoDisposition['attached_pic'] !== 1) {
+                $hasVideo = true;
```

Comment:
minor nit: You can add a `break;` after this to not iterate unnecessarily.

Reply:
thanks. done. https://github.com/vibhavsinha/PHP-FFMpeg/commit/7b572070ab499e9d39752bf7d19dc81f1679f8ce

Reply:
There is a missing closing brace in this fix. It opens 3 and only closes 2

Reply:
The code has been changed since the last comment here. I am sure it is balanced now. The reason for opening 3 and closing 2 is because the subtracted chunk also had one opening bracket. Hence, the added chunk should also have one extra opening brace.